tests/dtypes_test.py

```diff
@@ -64,6 +64,12 @@
 fp8_dtypes = [np.dtype(dtypes.float8_e4m3b11fnuz), np.dtype(dtypes.float8_e4m3fn),
               np.dtype(dtypes.float8_e4m3fnuz), np.dtype(dtypes.float8_e5m2),
               np.dtype(dtypes.float8_e5m2fnuz)]
+# TODO: Remove once minimum_jaxlib_version is 0.4.35+
+if jax.version._minimum_jaxlib_version >= "0.4.35":
```

Comment:
This condition means that the test is not run, even if a compatible jaxlib version is installed. Instead we should test against the actual installed jaxlib version; for example `if jaxlib.version.__version_info__ >= (0, 4, 35):`

Reply:
fixed. to avoid importing jaxlib I used `jax._src.lib.version`